pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -734,220 +734,224 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 
   app.controller('SectionAssessmentsController', ['$scope', '$routeParams', '$window', '$q', '$timeout', '$interval', '$sanitize', 'sectionsService', 'studentsService',
                                              function($scope, $routeParams, $window, $q, $timeout, $interval, $sanitize, sectionsService, studentsService) {
+    // Some strings.
+    var submission_list = {
```

Comment:
Good question.  Ideally probably, and properly processed/minified/etc, but we don't get that in Pegasus anyway.  This is how teacher dashboard was first built, so inclined to leave it like this for now.